src/main/PlagueMain.java

```diff
@@ -1032,8 +1036,8 @@ void resetRules() {
 
     }
 
-    String _leaderboardInit(int limit) {
-        return "[gold]no leaderboard cause i dont like the win based lb[white]";
+    String _leaderboardInit() {
+        return "This function should not be run. Leaderboard has been permanently disabled.";
```

Comment:
you might as well delete the entire function. Also, i want some kind of leaderboard at some point.